engines/scumm/he/moonbase/map_spiff.h

```diff
@@ -0,0 +1,95 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef SCUMM_HE_MOONBASE_MAP_SPIFF
+#define SCUMM_HE_MOONBASE_MAP_SPIFF
+
+#ifdef ENABLE_HE
+
+#include "engines/scumm/he/moonbase/map_mif.h"
+
+#define MAXELEVVAL 4 // for array size
+#define HIGH 3 // elevations
```

Comment:
I'd say these three constant names are prone to name clashes on some platforms. I'd recommend writing something lie `kElevHigh`

Reply:
Any update/thought on these?

Reply:
Somehow missed this.  Sorry.  This is done in commit 6222f86393c95802789673b1c955813687a46442.